app/assets/stylesheets/components/_auth-form.scss

```
@@ -0,0 +1,62 @@
+.auth-form {
```

How about using the UI element type as a prefix to align with [our convention](https://nimblehq.co/compass/development/code-conventions/css/#convention-25)?

```suggestion
.form-auth {
```

---

Yup! Updated in https://github.com/aetheryna/google-scraper-ruby/pull/32/commits/8be084d31be5d99dcff285a79d4d1db9531d6ac0 😄